components/Note.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import Link from 'next/link'
+
+import { Note } from '../interfaces'
+
+type Props = {
+data: Note
+remove: any,
+seek: any,
+}
+
+function to_hms(t: number) {
+   let str = new Date(t * 1000).toISOString();
+   if(t < 60) {
+      return str.substr(17, 6);
```

Comment:
even without `moment`, if you leave this as a `Date` object you could just use methods like `Date.getSeconds()` rather than substrings. [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) is your friend

Reply:
Fixed to use getUTCSeconds() and friends